api-docs/client-files.md

```diff
@@ -0,0 +1,262 @@
+# Client files
+
+This api is provided to upload and keep track of client files.
+
+## Get a single client file
+
+```http
+GET /api/clients/:file/:id/files
+```
+
+### Response
+
+#### Success
+
+```json
+{
+  "createdBy": {
+    "firstName": "Sean",
+    "lastName": "White",
+    "fullName": "Sean White",
+    "timestamp": "2019-11-06T06:00:00.000Z"
+  },
+  "lastUpdatedBy": {
+    "firstName": "Sean",
+    "lastName": "White",
+    "fullName": "Sean White",
+    "timestamp": "2019-11-06T06:00:00.000Z"
+  },
+  "fileName": "file-1",
+  "filePath": "file://host/path"
```

Comment:
What is this filePath going to be? Strings that start with `file://` usually refer to files that are local to the computer. For the user of this API, they won't have this file on their hard drive. Is `filePath` the URL to download the file? If we're using signed s3 urls, I might even expect a completely separate api request to get the signed url to download the file (instead of always creating a signed url to download even when you're just viewing the file list).